pkgs/data/icons/gruvbox-dark-icons-gtk/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "gruvbox-dark-icons-gtk";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "jmattheis";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1fks2rrrb62ybzn8gqan5swcgksrb579vk37bx4xpwkc552dz2z2";
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/icons/
```

Install to `oomox-gruvbox-dark`, which is the name of the icon theme according to the `index.theme` file.
```suggestion
    mkdir -p $out/share/icons/oomox-gruvbox-dark
```